netci.groovy

```diff
@@ -1789,6 +1800,13 @@ Constants.allScenarios.each { scenario ->
                         os = 'Windows_NT'
                     }
 
+                    def validWindowsNTCrossArches = ["arm", "armlb", "arm64"]
+                    def isWindowsArmJob = (os == "Windows_NT" && architecture in validWindowsNTCrossArches)
+
+                    if (scenario == 'pri1' && isWindowsArmJob == false) {
```

Comment:
This code should be moved below, along with the rest of the per-scenario checking.

Reply:
Actually, I think we should get rid of this code. We should allow pri1 jobs to exist and be triggerable for all os/architectures. Currently, 'default' is always pri-0, and is named "Innerloop". We should make scenario 'pri1' do full pri-1 builds and test runs, and be triggerable using exactly the same name but without "Innerloop".

So, e.g., in addition to the 'default' auto-triggered:
```
Windows_NT x64 Checked Innerloop Build and Test
```
that does pri-0 testing, we should have:
```
Windows_NT x64 Checked Build and Test
```
that does pri-1 testing, and is not auto-triggered, but does have a trigger phrase.

This would require a little more work, e.g. in the addTriggers arm/Windows_NT case:
```
if (scenario == 'default') {
    contextString += " Innerloop"
    triggerString += "\\W+Innerloop"
}
else {
    contextString += " ${scenario}"
    triggerString += "\\W+${scenario}"
}
```
would become:
```
if (scenario == 'default') {
    contextString += " Innerloop"
    triggerString += "\\W+Innerloop"
}
else if (scenario == 'pri1') {
    // pri1 has an empty scenario string
}
else {
    contextString += " ${scenario}"
    triggerString += "\\W+${scenario}"
}
```

Or, we could just let the string be:
```
Windows_NT x64 Checked pri1 Build and Test
```
but that seems a tad ugly.

Reply:
This could be a separate PR.